File: exam-slammr-webapp/src/test/java/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/user/UserConnectionItemTest.java
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.user;

import org.junit.Test;

import uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.user.UserConnectionItem;

import nl.jqno.equalsverifier.EqualsVerifier;

/**
 * Unit test class for {@link UserConnectionItem}
 */
public class UserConnectionItemTest {

	@Test
	public void shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(UserConnectionItem.class)
				.verify();
	}
}

File: exam-slammr-webapp/src/test/kotlin/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/user/UserConnectionItemTest.kt
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.user

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.Test

/**
 * Unit test class for [UserConnectionItem]
 */
class UserConnectionItemTest {

	@Test
	fun shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(UserConnectionItem::class.java)
				.verify()
	}
}